nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/RepositoryElasticSearchFeature.java

```diff
@@ -36,8 +37,9 @@
 // TODO: "org.nuxeo.ecm.core.management" is missing but creates test failures
 @Deploy({"org.nuxeo.runtime.jtajca", "org.nuxeo.ecm.core.io", "org.nuxeo.ecm.automation.server",
         "org.nuxeo.ecm.automation.io", "org.nuxeo.ecm.webengine.core", "org.nuxeo.ecm.webengine.jaxrs",
-        "org.nuxeo.ecm.platform.web.common", "org.nuxeo.elasticsearch.core", "org.nuxeo.ecm.platform.query.api"})
+        "org.nuxeo.ecm.platform.web.common", "org.nuxeo.elasticsearch.core", "org.nuxeo.ecm.platform.query.api", "org.nuxeo.ecm.core.management" })
```

Comment:
Thanks this fix the pb for elasticsearch feature, still the output of the t&p contains dozen of core management component missing in other modules.
I don't really see the point of deploying probes by default it is just one more deps for tests.